.github/workflows/odh-manifests-PR-sync.yml

```diff
@@ -0,0 +1,70 @@
+name: odh-manifests sync
+
+run-name: Sync manifests in odh-manifests
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  send-pull-requests:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout data-science-pipelines-operator repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: main
+          repository: DharmitD/data-science-pipelines-operator
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Send pull-request
+        run: |
+          LATEST_TAG=$(git describe --tags --always --abbrev=0)
+          REPOSITORY="DharmitD/odh-manifests"
+          FOLDER="bin/$REPOSITORY"
+          BRANCH_NAME="chore-update-scripts-to-$LATEST_TAG"
+
+          # Clone the remote repository and change working directory to the
+          # folder it was cloned to.
+          git clone \
+            --depth=1 \
+            --branch=master \
+            https://DharmitD:${{ secrets.ACCESS_TOKEN }}@github.com/$REPOSITORY \
+            $FOLDER
+          echo "Access token value: ${{ secrets.ACCESS_TOKEN }}"
+          cd $FOLDER
+
+          # Setup the committers identity.
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git config user.name "GitHub Actions"
+
+          # Create a new feature branch for the changes.
+          git checkout -b $BRANCH_NAME
+          echo "Created branch: $BRANCH_NAME"
+
+          # Copy DSPO manifests. Using rsync to allow filtering of paths/files (e.g. like a .gitignore, hidden files, etc)
+          rsync -av --delete ../../../config/ data-science-pipelines-operator/
+
+          # Commit the changes and push the feature branch to origin
+          git add .
+          git commit -m "Update DSP Operator manifests to $LATEST_TAG"
+          # Check if the branch exists and perform rebase if it does
+          if git ls-remote --exit-code --heads origin $BRANCH_NAME; then
+            git pull --rebase origin $BRANCH_NAME
+          fi
+          git push origin $BRANCH_NAME
+
+          # Store the PAT in a file that can be accessed by the
+          # GitHub CLI.
+          echo "${{ secrets.ACCESS_TOKEN }}" > token.txt
+
+          # Authorize GitHub CLI for the current repository and
+          # create a pull-requests containing the updates.
+          gh auth login --with-token < token.txt
```

Comment:
You can provide the token for `gh` using enviroment variables, `gh` automatically picks up `GITHUB_TOKEN`:
```
      env:
        GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```